modules/10-basics/20-forms/description.en.yml

```diff
@@ -1,27 +1,27 @@
 ---
 
-name: Формы
+name: Forms
 theory: |
-  Про Lisp-подобные языки говорят, что у этих языков нет синтаксиса. Синтаксис у них конечно есть, но максимально примитивный, фактически состоящий из списков и значений. Кроме того, в лиспах отсутствуют ключевые слова и соответствующие им конструкции. В обычных языках существует множество управляющих конструкций, таких как условия, циклы, возврат, присвоение переменных и многое другое. В лиспоподобных языках таких конструкций нет (это не значит, что на Clojure нельзя реализовать цикл или написать условие — можно!)
+  Lisp-like languages are said to be languages without syntax. Of course, they have a syntax, but it is as primitive as possible, actually consisting of lists and values. In addition, Lisp languages do not have keywords and their corresponding constructs. In popular languages, there are many control constructions, such as conditions, loops, returns, variable assignment, and more. Lisp-like languages have no such constructs (this does not mean that you can't implement a loop or write a condition in Clojure!)
 
-  Каким же образом Clojure понимает, с чем сейчас он работает и что нужно делать? Все дело в _формах_. Любая корректная программа на Lisp называется _формой_. Например:
+  So how does Clojure know what it's working with now and what to do? It's all about _forms_. Any correct Lisp program is called a _form_. For example:
 
   ```clojure
-  ;формы
+  ;forms
   (println "some-str")
   (+ 1 2 3)
   (- 3)
   8
   "hello world!"
 
-  ; а это не форма, такой код завершится ошибкой
+  ; not forms, program will fail with error
   (1 2)
   Execution error (ClassCastException) at user/eval1 (REPL:1).
   class java.lang.Long cannot be cast to class clojure.lang.IFn (java.lang.Long is in module java.base of loader 'bootstrap'; clojure.lang.IFn is in unnamed module of loader 'app')
   ```
 
-  Форм всего две — нормальная и составная. Нормальной соответствуют все значения (и определения, с которыми мы познакомимся позже), так как они вычисляются сами в себя, например, число _8_ или строка "hello". Составная форма — это список, который нужно обработать тем или иным способом (вычислить).
+  There are only two forms: normal and composite. The normal form corresponds to all values (and definitions, which we will get to know later) because they are computed in themselves, e.g., the number _8_ or the string "hello". A compound form is a list that needs to be processed (computed) in one way or another.
```

Comment:
they are computed in themselves => they evaluate to themselves
processed (computed) => processed (evaluated)